Doc/library/argparse.rst

```diff
@@ -1429,6 +1417,20 @@ this API may be passed as the ``action`` parameter to
       and return a string which will be used when printing the usage of the program.
       If such method is not provided, a sensible default will be used.
 
+.. class:: BooleanOptionalAction
+
+   A subclass of :class:`Action` for handling boolean flags with positive
+   and negative options. Adding a single argument such as ``--foo`` automatically
+   creates both ``--foo`` and ``--no-foo`` options, storing ``True`` and ``False``
+   respectively::
+
+       >>> import argparse
+       >>> parser = argparse.ArgumentParser()
+       >>> parser.add_argument('--foo', action=argparse.BooleanOptionalAction)
+       >>> parser.parse_args(['--no-foo'])
+       Namespace(foo=False)
+
+   .. versionadded:: 3.9
 
```

Comment:
Please add a blank line here. There were two blank lines between sections.

Reply:
fixed. Just wondering:
- is this a convention in reST? Maybe I missed this, but I can't find any relevant description in the [devguide](https://devguide.python.org/documentation/markup/#paragraphs). 
- is there a linter that can detect this?

Reply:
Appreciated!